wallet/chainntfns.go

```diff
@@ -280,11 +280,19 @@ func (w *Wallet) addRelevantTx(dbtx walletdb.ReadWriteTx, rec *wtxmgr.TxRecord,
 	// relevant.  This assumption will not hold true when SPV support is
 	// added, but until then, simply insert the transaction because there
 	// should either be one or more relevant inputs or outputs.
-	err := w.TxStore.InsertTx(txmgrNs, rec, block)
+	exists, err := w.TxStore.InsertTxCheckIfExists(txmgrNs, rec, block)
 	if err != nil {
```

Comment:
Typo in commit message InserTxCheckIfExists -> InsertTxCheckIfExists

Also commit body should be wrapped to 80 chars

Reply:
Changed!